conf/app.routes

```diff
@@ -5,6 +5,8 @@ GET   /language/:lang                    controllers.SwitchLanguageController.sw
 
 GET   /accessibility                     controllers.AccessibilityStatementController.getPage
 
+GET   /:processCode/session-timeout      controllers.SessionTimeoutPageController.getPage(processCode: String)
```

Comment:
changed mind on this, /session-timeout/:processCode follows the accessibility out-of-guidance url pattern. Using /:processCode/session-timeout limits the possible urls within a piece of guidance unecessarilly.

Reply:
Have posted on design channel to see which they prefer